includes/country-functions.php

```diff
@@ -476,6 +479,38 @@ function edd_get_states_list() {
 
 	return apply_filters( 'edd_us_states', $states );
 }
+/**
+ * Get Angola States
+ *
+ * @since 1.1
+ * @return array $states A list of states
+ */
+function edd_get_angola_list() {
+	$states = array(
+		''   => '',
+	   	'BGO' => __( 'Bengo', 'edd' ),
```

Comment:
Can the formatting be fixed up here please - to ensure tabs are used and not spaces.